packages/datetime/src/dateRangePicker.tsx

```diff
@@ -58,6 +58,15 @@ export interface IDateRangePickerProps extends IDatePickerBaseProps, IProps {
      * @default true
      */
     contiguousCalendarMonths?: boolean;
+    /**
```

Comment:
nit: add newline above. I'll fix this after merging.

Reply:
Also might be time to pull out some of these shared props into a common interface like `IDateRangePickerCoreProps` or something, so we don't have to write the prop descriptions in two places. Again, no action needed now.

Reply:
yeah def worth making this `dayPickerProps` sharable cuz the docs are identical across usages.